library/firewall_lib.py

```diff
@@ -364,6 +364,11 @@ def main():
                 icmp_block,
             )
         )
+    
+    if not (HAS_FIREWALLD_NM and nm_is_imported()) and (
```

Comment:
nm support was removed here - https://github.com/linux-system-roles/firewall/pull/32/commits/a6e427e5983185fbb54c3b5b036dd3bf48bb693f - this also got rid of `HAS_FIREWALLD_NM` and `nm_is_imported` - so do we still need this PR?  Should it instead use `HAS_FIREWALLD`?